README.rst

```diff
@@ -115,6 +115,11 @@ Steps:
 5) Deploy truffle project onto (local private) Ethereum chain
    ``truffle migrate``
 6) Install python dependencies ``pip3 install -r requirements.txt``
+   + It has been a common practice to setup a python environment with
```

Comment:
The extra plus in this line causes it to not be a part of the quote block below the step.